src/main/webapp/assets/modules/current-job/salary-review.con.js

```diff
@@ -181,4 +182,8 @@ techlooper.controller("salaryReviewController", function ($scope, $rootScope, js
   $scope.createNewReport = function () {
     $route.reload();
   }
+
+  $scope.openFacebookShare = function() {
+    window.open('https://www.facebook.com/sharer/sharer.php?u=' + baseUrl + '/renderSalaryReport/' + $rootScope.language + '/' + $scope.salaryReview.createdDateTime, 'name','width=450 ,height=350');
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Too many errors. (98% scanned).